contracts/DelayView.sol

```diff
@@ -56,21 +56,27 @@ contract DelayView is DelayManageable {
     function GetMinDelays(address _token, uint256 _amount)
         public
         view
-        returns (uint256 _startDelay, uint256 _finishDelay)
+        returns (
+            uint256 _startDelay,
+            uint256 _cliffDelay,
+            uint256 _finishDelay
+        )
     {
         if (
             DelayLimit[_token].Amounts.length == 0 ||
```

Comment:
Make a new "memory" variable for `DelayLimit[_token]`. This way we read from storage only once.